.circleci/config.yml

```diff
@@ -125,14 +111,15 @@ jobs:
 
              for PKG in $PKGS ; do
                  echo_blue "===+ Building: $PKG"
-                 make "package/$PKG/compile" -j3 V=s
+                 # compiles as fast or verbose if failed on first try
+                 make "package/$PKG/compile" -j3 || make "package/$PKG/compile" V=s
```

Comment:
It might hide // compilation issue, so not a fan
we can remove V=s if you want, but I'm curious how much speed we gain

Reply:
Removed it.